global/utils/proxyUtils.ts

```diff
@@ -6,3 +6,7 @@ const convertToRegexPath = (path: string) => {
 export const removeFromPath = (url: string, path: string) => {
 	return url?.replace(convertToRegexPath(path), '') || '';
 };
+
+// Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections
+// and HTTPS requests insecure by disabling certificate verification
+export const SSLSecured = Number(process.env.NODE_TLS_REJECT_UNAUTHORIZED) != 0;
```

Comment:
🤔 A more strict check should prevent `false`, `Null` and empty strings being matched as if intending to disable this (as those values are [coerced to 0](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_coercion)).

```suggestion
export const SSLSecured = !(Number(process.env.NODE_TLS_REJECT_UNAUTHORIZED) === 0);
```